datafusion/core/src/dataframe/mod.rs

```diff
@@ -225,7 +225,12 @@ impl DataFrame {
     /// # async fn main() -> Result<()> {
     /// let ctx = SessionContext::new();
     /// let df = ctx.read_csv("tests/data/example.csv", CsvReadOptions::new()).await?;
-    /// let df = df.select_columns(&["a", "b"])?;
+    /// df.select_columns(&["a", "b"])?.show().await?;
```

Comment:
What would you think about using [`assert_batches_eq` ](https://docs.rs/datafusion/latest/datafusion/macro.assert_batches_eq.html)instead so that the output is automatically checked as part of the doc tests rather than relying on us manually keeping it up to date?

It is of similar readability I think. Here is an example:

https://github.com/apache/datafusion/blob/d9cb6e677ec437a9c6ff2cc0900f902804da39bb/datafusion/core/tests/sql/select.rs#L27-L49

Reply:
My original thought is to simplify large long column on doctest(less readable), like this: https://github.com/Eason0729/datafusion/blob/4dd44c5a2b0d9810d7e9163689afab227c58d542/datafusion/core/src/dataframe/mod.rs#L734

Therefore, I simplify example_long.csv in the next commit to the point which is just enough to showcase most method on dataframe.

Reply:
But the expect behavior on `describe` method is more complex, which might require dedicated csv, so I decided to leave as it is.